lib/actions/model.js

```diff
@@ -108,4 +108,11 @@ module.exports = {
       return validationErrors;
     };
   },
+
+  set(model, ...args) {
+    return () => {
+      model.set(...args);
```

Comment:
From a `brainstem-js` perspective, this is not utilizing the storage manager whatsoever. If it doesn't need the storage manager, it should not belong in `brainstem-redux`. 

`brainstem-redux` is a library which interfaces the storage manager -- specifically the data it stores -- for usage with Redux (`mapStateToProps`, `mapDispatchToProps`, reducers). This abstraction avoids the necessity of using backbone models and collections in our React+Redux ecosystem.

As for what I think a good design for "set" is:

Set should take a model type, an ID, and the new set of attributes. These three things can be used to look up the model in the storage manager and update its attributes.

We already have a [helper method which updates the model in the storage manager](https://github.com/mavenlink/brainstem-redux/blob/master/lib/reducers/update-model.js). I think this is used to reduce an action.